mod_reforged/hooks/config/strings.nut

```diff
@@ -900,7 +907,7 @@ foreach (vanillaDesc in vanillaDescriptions)
 			Type = ::UPD.EffectType.Passive,
 			Description = [
 				"[Resolve|Concept.Bravery] is increased by " + ::MSU.Text.colorPositive("10%") + " at all times.",
-				"Additionally, at positive morale checks, [Resolve|Concept.Bravery] is increased by a further " + ::MSU.Text.colorPositive(10) + ".",
+				"Additionally, for positive morale checks, [Resolve|Concept.Bravery] is increased by a further " + ::MSU.Text.colorPositive("10%") + ".",
```

Comment:
This is minor but `at` morale checks is vanilla convention e.g. in superstitious trait. So perhaps we should stick with that just for consistency across tooltips (OCD :P) unless we want to change vanilla to `for` too .

Reply:
at is fine